eval/src/vespa/eval/eval/simple_value.cpp

```diff
@@ -45,6 +46,10 @@ struct CreateValueFromTensorSpec {
             }
             map[sparse_key][dense_key] = entry.second;
         }
+        // hack for passing some (invalid?) unit tests
```

Comment:
not really a hack since converting from a spec should use 0.0 as default value for all existing non-defined cells. good catch though.